app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+  background-image: url(image_path("welcome.jpg"));
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: table;
+  min-height: 500px;
+  width: 100%;
+  .banner_content {
+		display: table-cell;
+		text-align: center;
+		vertical-align: middle;
+		h1 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented with spaces, not tabs<br>Selector should have depth of applicability no greater than 2, but was 3